themes/digital.gov/src/scss/new/_card.scss

```diff
@@ -169,3 +169,26 @@
   padding: units(7);
   text-align: center;
 }
+
+// topics page card-legislation
+.dg-card-legislation {
+  .usa-card-group {
+    @include at-media("desktop") {
+      left: 0.55rem;
+      margin: 0;
+      position: relative;
+    }
+
+    .usa-card__img {
+      background-color: white;
+    }
+
+    .usa-card__heading {
+      font-size: 1.1rem;
```

Comment:
You can use `md` token here:

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/9db66ce3-474d-4015-b36a-de70d6d54bb7)

Reply:
Roger that, updated to use token.